OpenMage/magento-mirror
# Updating to pristine 1.9.3.10 version

I don't think I created the tag correctly. Happy to fix that if you let me know what the process is.

Here's what I've done
1. Download pristine 1.9.3.10 from https://magento.com/tech-resources/download
2. Forked the OpenMage repo (actually I already had a fork, so I did `git fetch upstream` instead)
3. `rm -rf`ed all files in the root
4. Did a `git status | grep deleted` to find and delete files that were removed between `1.9.3.9` and `1.9.3.10`.  
5. `git add . && git commit -m "Updating to pristine 1.9.3.10 version" && git push`

Testing procedure:
1. Download the zipped version of my commit from GitHub
2. Extract pristine files downloaded from Magento into a separate dir
3. `diff -r ~/Downloads/magento-mirror-magento-1.9 ~/Downloads/magento-1.9.3.10-pristine/`
4. Observed that `diff` found no changes.
5. `echo "testing that diff works" >> RELEASE_NOTES.txt`
6. Run diff command again, observe that it finds this deliberate change (just to confirm that diff isn't broken)

How I created the tag (probably wrong; please advise):
1. `git tag 1.9.3.10`
2. `git push origin 1.9.3.10`
3. Created this pull request.
